### ISSUE TITLE
No trade & coin_list

### ISSUE THREAD

**Issue by jdk-sal3** [mdabessa/trade_bot#2]
I found the following:

1. With default coint_list, it only show "Scouting initial prices..." and not making any trade for hours now
2. If I edit the coin_list.txt, it will give me error:

Traceback (most recent call last):
  File "/home/ubuntu/trade_bot/main.py", line 31, in <module>
    manager = ManagerBinance(
  File "/home/ubuntu/trade_bot/modules/manager.py", line 251, in __init__
    price = float(self.client.get_margin_price_index(symbol=pair)["price"])
  File "/home/ubuntu/trade_bot/env/lib/python3.10/site-packages/binance/client.py", line 3300, in get_margin_price_index
    return self._request_margin_api('get', 'margin/priceIndex', data=params)
  File "/home/ubuntu/trade_bot/env/lib/python3.10/site-packages/binance/client.py", line 364, in _request_margin_api
    return self._request(method, uri, signed, **kwargs)
  File "/home/ubuntu/trade_bot/env/lib/python3.10/site-packages/binance/client.py", line 315, in _request
    return self._handle_response(self.response)
  File "/home/ubuntu/trade_bot/env/lib/python3.10/site-packages/binance/client.py", line 324, in _handle_response
    raise BinanceAPIException(response, response.status_code, response.text)
binance.exceptions.BinanceAPIException: APIError(code=-3042): priceIndex not available for this margin pair.
 
I have tried to reset the database using the reset_database.py, but it still showing this error. 
The error is gone after I re-download the coin_list.txt.

Any workaround?

**Comment by mdabessa** [comment]
1. The bot just starts trading when fills the database with historic data, where by default it takes 1 day.
2. What coins are you using?, the coins in the coin_list.txt need to pair with USDT on Binance.

**Comment by jdk-sal3** [comment]
Thanks for your reply

1. I will wait for 24 hours, then. I have it running for hours and pass a new day on server time and I have new msg: Date: 2022-04-07 00:00:29, but still no initial trades. Waiting few more hours....
2. I only remove some coins from the original list without adding new ones. Even when I (only) remove the # from the LTC, it gave me the error.

I have tried this bot different machines but ending up with the same error. Perhaps my setup is wrong? This is how I set it up on fresh oracle cloud instance, also on WSL2, ubuntu20:
- install python 3.10, set as default
- install postgres v12 (also tried v14), create database, gave authorization
- clone repo, create .env, install requirements
Are these steps correct?
